### PR TITLE
8299227: host `exif.org` not found in link in doc comment

### PR DESCRIPTION
**Pull request by kumarabhi006**
Broken links are replaced by the similar links fixed in [JDK-8288527](https://bugs.openjdk.org/browse/JDK-8288527) for file `src/java.desktop/share/classes/javax/imageio/plugins/tiff/ExifTIFFTagSet.java`.

---------
### Progress
- [ ] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8299227](https://bugs.openjdk.org/browse/JDK-8299227): host `exif.org` not found in link in doc comment

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11768/head:pull/11768` \
`$ git checkout pull/11768`

Update a local copy of the PR: \
`$ git checkout pull/11768` \
`$ git pull https://git.openjdk.org/jdk pull/11768/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11768`

View PR using the GUI difftool: \
`$ git pr show -t 11768`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11768.diff">https://git.openjdk.org/jdk/pull/11768.diff</a>

</details>

### CODE REVIEW

**Comment by jayathirthrao** [comment]
Since this is a simple change we can still push it to JDK20.

**Comment by prrace** [comment]
The suggestion was to fix this doc bug in JDK20. So this is the wrong repo.
You need to withdraw this PR and resubmit against JDK 20
Remember JDK 20 fixes are forward ported to JDK mainline, not the other way around.

File: src/java.desktop/share/classes/com/sun/imageio/plugins/tiff/TIFFImageWriteParam.java [hunk 1]
```diff
@@ -78,7 +78,7 @@
  * <tr>
  * <td>Exif JPEG</td>
  * <td>Exif-specific JPEG compression (see note following this table)</td>
- * <td><a href="http://www.exif.org/Exif2-2.PDF">Exif 2.2 Specification</a>
+ * <td><a href="https://www.cipa.jp/std/documents/e/DC-008-2012_E.pdf">Exif 2.2 Specification</a>
```
Comment (prrace) [inline src/java.desktop/share/classes/com/sun/imageio/plugins/tiff/TIFFImageWriteParam.java:81]: BTW the link at the other end is EXIF 2.3 .. so you probably want to fix the text too.

File: src/java.desktop/share/classes/com/sun/imageio/plugins/tiff/TIFFImageWriteParam.java [hunk 2]
```diff
@@ -78,7 +78,7 @@
...
  * (PDF), section 4.5.5, "Basic Structure of Thumbnail Data"</td>
```
Comment (prrace) [inline src/java.desktop/share/classes/com/sun/imageio/plugins/tiff/TIFFImageWriteParam.java:82]: This is now 4.5.8

File: src/java.desktop/share/classes/javax/imageio/metadata/doc-files/tiff_metadata.html [hunk 3]
```diff
@@ -574,7 +574,8 @@ <h3><a id="Compression">Compression</a></h3>
 <th scope="row">9</th>
 <td>Exif JPEG</td>
 <td>Exif-specific JPEG compression (see note following this table)</td>
-<td><a href="http://www.exif.org/Exif2-2.PDF">Exif 2.2 Specification</a>
+<td><a href="https://www.cipa.jp/std/documents/e/DC-008-2012_E.pdf">
+    Exif 2.2 Specification</a>
 (PDF), section 4.5.5, "Basic Structure of Thumbnail Data"</td>
```
Comment (prrace) [inline src/java.desktop/share/classes/javax/imageio/metadata/doc-files/tiff_metadata.html:579]: This is now 4.5.8

File: src/java.desktop/share/native/libjavajpeg/imageioJPEG.c [hunk 4]
```diff
@@ -1614,7 +1614,7 @@ Java_com_sun_imageio_plugins_jpeg_JPEGImageReader_setSource
 /*
  * For EXIF images, the APP1 will appear immediately after the SOI,
  * so it's safe to only look at the first marker in the list.
- * (see http://www.exif.org/Exif2-2.PDF, section 4.7, page 58)
+ * (see https://www.cipa.jp/std/documents/e/DC-008-2012_E.pdf, section 4.7, page 58)
  */
 #define IS_EXIF(c) \
```
Comment (prrace) [inline src/java.desktop/share/native/libjavajpeg/imageioJPEG.c:1619]: Page number is no longer correct

File: src/java.desktop/share/native/libjavajpeg/imageioJPEG.c [hunk 5]
```diff
@@ -1715,7 +1715,7 @@ Java_com_sun_imageio_plugins_jpeg_JPEGImageReader_readImageHeader
              *  - we got JFIF image
              *     Must be YCbCr (see http://www.w3.org/Graphics/JPEG/jfif3.pdf, page 2)
              *  - we got EXIF image
-             *     Must be YCbCr (see http://www.exif.org/Exif2-2.PDF, section 4.7, page 63)
+             *     Must be YCbCr (see https://www.cipa.jp/std/documents/e/DC-008-2012_E.pdf, section 4.7, page 63)
```
Comment (prrace) [inline src/java.desktop/share/native/libjavajpeg/imageioJPEG.c:1718]: Page number is no longer correct

**Review by prrace — CHANGES_REQUESTED** [review]

File: src/java.desktop/share/native/libjavajpeg/imageioJPEG.c (hunk 4 above)
Comment (prrace) [reply src/java.desktop/share/native/libjavajpeg/imageioJPEG.c:1619]: I think it should be page 83

**Comment by kumarabhi006** [comment]
I will raise a PR for JDK20 repo with suggested changes.
Thank you.

**Comment by kumarabhi006** [comment]
Withdrawing this PR.
I will submit for JDK20 repo.

**Pull request closed without merging**